setup.py

```diff
@@ -64,7 +64,7 @@ def running_under_virtualenv():
         if distro == 'Ubuntu':
             data_files.append(('/etc/init',
                                ['debian/upstart/diamond.conf']))
-        if distro in ['centos', 'redhat', 'debian', 'fedora']:
+        if distro in ['centos', 'redhat', 'debian', 'fedora', 'system']:
```

Comment:
Yeah, I agree, it's pretty wacked.   However, we've been running diamond on
amazon linux with this patch, so it at least solves that problem...

On Sun, Jan 4, 2015 at 9:32 PM, Rob Smith <notifications@github.com> wrote:

> In setup.py
> <https://github.com/BrightcoveOS/Diamond/pull/770#discussion-diff-22446745>
> :
>
> > @@ -64,7 +64,7 @@ def running_under_virtualenv():
> >          if distro == 'Ubuntu':
> >              data_files.append(('/etc/init',
> >                                 ['debian/upstart/diamond.conf']))
> > -        if distro in ['centos', 'redhat', 'debian', 'fedora']:
> > +        if distro in ['centos', 'redhat', 'debian', 'fedora', 'system']:
>
> the fact that they report themselves as system, that really bugs me :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BrightcoveOS/Diamond/pull/770/files#r22446745>.
>